### ISSUE TITLE
Release Announcements block interactions in modals without being able to be closed via keyboard

### ISSUE THREAD

**Issue by t3chguy** [element-hq/element-web#27443]
### Steps to reproduce

1. Log into a new account such that you get the TAC release announcement
2. Open user settings or some other large modal
3. Observe that you cannot use Tab to reach the TAC RA to dismiss it to be able to see the content underneath it (modal)

#### What did you expect?

Either the RA renders underneath or it is closeable via keyboard

#### What happened instead?

It renders on top but does not appear in tabstop list

https://github.com/element-hq/element-web/assets/2403652/36ca986f-1c7f-4d89-98f4-796bd91b7ba0

### Application version

1.11.66

### Will you send logs?

No
